cmsplugin_cascade/bootstrap3/buttons.py

```diff
@@ -19,7 +18,7 @@ class ButtonTypeRenderer(RadioFieldRenderer):
     """
     Render sample buttons in different colors in the button's backend editor.
     """
-    BUTTON_TYPES = SortedDict((('btn-default', _('Default')), ('btn-primary', _('Primary')),
+    BUTTON_TYPES = dict((('btn-default', _('Default')), ('btn-primary', _('Primary')),
```

Comment:
Why do you replace the ``SortedDict`` against a dict? We then lose the ordering.
If, then change this to ``collections.OrderedDict``.